frontend/src/config/mongolia/layers.json

```diff
@@ -4,7 +4,7 @@
     "path": "data/mongolia/admin_boundaries.json",
     "opacity": 0.8,
     "admin_code": "NSO_CODE",
```

Comment:
@wadhwamatic then this should likely be updated as well no?

Reply:
I took a guess when I copied `admin_code` into `admin_level_codes` here, but I was a little puzzled by the fact that the format of codes did not seem consistent with other countries (5 digits vs 6).